metpy/deprecation.py

```diff
@@ -0,0 +1,316 @@
+# Copyright (c) 2008-2017 MetPy Developers.
```

Comment:
Might be better to blacklist the license check for this file in `setup.cfg` rather than, essentially, an incorrect block before the proper one. It could then go back to being a comment.